src/styles/mixins/use-logical.scss

```diff
@@ -0,0 +1,48 @@
+@use "../functions";
+
+@function getLogicalName ($property, $str) {
+  $splittedProperty: functions.str-split($property, '-');
+  $propertyHead: nth($splittedProperty, 1);
+  $propertyTail: nth($splittedProperty, 2);
+  @return #{$propertyHead}#{$str}#{$propertyTail};
+};
+
+@function getLogicalValues($values) {
+  $results: [];
+  @for $index from 1 through 4 {
+    $valueIndex: $index;
+    @if $index > length($values) {
+      @if $index == 2 {
+        $values: append($values, nth($values, 1));
+      }@else {
+        $valueIndex: $index - 2;
+      };
+    }
+    $value: #{nth($values, $valueIndex)};
+    $results: append($results, $value);
+  }
+  @return $results
+}
+
+@mixin use-logical($element, $property, $values) {
+  @if $values and $property {
+    $includeBorderRadius: str-index($property, border-radius);
+    @if $includeBorderRadius {
```

Comment:
@ArkadiuszKrzymienski tips from my side. Maybe `@mixin` should be the easiest to read. 

```
@if $includeBorderRadius {
  @mixin borderRadiusLogicalProperties
}
@else {
  @mixin defaultLogicalProperties
}
```
but now it's fine :) 
and maybe in SCSS we should call variables like `include-border-radius` and `logical-values` etc.